src/site/includes/appeals-banner.html

```diff
@@ -0,0 +1,15 @@
+  <div class="usa-alert-full-width usa-alert-full-width-warning">
+    <div aria-live="polite" role="alert" class="usa-alert usa-alert-warning">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">The Appeals process has changed</h3>
+        <div class="usa-alert-text">
+          <b>For VA decisions received aftger February 19, 2019</b>, please keep reading below to learn about your decision review options.
```

Comment:
It looks like "after" has an extra "g".

What do you think about wrapping the individual text blocks in `<p>` tags, and removing the `<br>` tags? It changes the spacing between blocks a touch, so please take that as a question to consider only.

Reply:
Thanks for spotting that stay "g".

For the spacing, it's mostly how I've been handling spacing for the .md files. 

I attempted your advice and wrapped the text in a `<p>`, but it wasn't giving me the line breaks and spacing as intended, so put the `<br>`'s back in.